hazelcast/src/main/java/com/hazelcast/config/MapConfig.java

```diff
@@ -886,6 +893,46 @@ public MapConfig setHotRestartConfig(HotRestartConfig hotRestartConfig) {
         return this;
     }
 
+    /**
+     * Gets the {@code MerkleTreeConfig} for this {@code MapConfig}
+     *
+     * @return merkle tree config
+     */
+    public MerkleTreeConfig getMerkleTreeConfig() {
+        return merkleTreeConfig;
+    }
+
+    /**
+     * Sets the {@code MerkleTreeConfig} for this {@code MapConfig}
+     *
+     * @param merkleTreeConfig merkle tree config
+     * @return this {@code MapConfig} instance
+     */
+    public MapConfig setMerkleTreeConfig(MerkleTreeConfig merkleTreeConfig) {
```

Comment:
is a `null` argument allowed here?

Reply:
Probably not. `null` will bring just problems somewhere down the line. I'll add a check.